fybrik/README.md

```diff
@@ -0,0 +1,47 @@
+# Accessing a Dataset by a Fybrik Application
+
+We explain how to access a dataset through the Airbyte Module, by a Fybrik application. In this example, we use the `letter-frequency` dataset, a CSV file found in https://people.sc.fsu.edu/~jburkardt/data/csv/letter_frequency.csv.
+
+You will need a copy of the Fybrik repository (`git clone https://github.com/fybrik/fybrik.git`). Set the following environment variables: FYBRIK_DIR for the path of the `fybrik` directory, and AIRBYTE_MODULE_DIR for the path of the `airbyte-module` directory.
+
+1. Install Fybrik Prerequisites. Follow the instruction in the Fybrik [Quick Start Guide](https://fybrik.io/dev/get-started/quickstart/). Stop before the "Install control plane" section.
+
+1. Before installing the control plane, we need to customize the [Fybrik taxonomy](https://fybrik.io/dev/tasks/custom-taxonomy/), to define new connection and interface types. Run:
+    ```bash
+    cd $FYBRIK_DIR
+    go run main.go taxonomy compile --out custom-taxonomy.json --base charts/fybrik/files/taxonomy/taxonomy.json $AIRBYTE_MODULE_DIR/fybrik/fybrik-taxonomy-customize.yaml
+    helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
+    helm install fybrik charts/fybrik --set global.tag=master --set global.imagePullPolicy=Always -n fybrik-system --wait --set-file taxonomyOverride=custom-taxonomy.json
+    ```
+
+1. Install the Airbyte module:
+    ```bash
+    kubectl apply -f $AIRBYTE_MODULE_DIR/module.yaml -n fybrik-system
+    ```
+
+1. Create a new namespace for the application, and set it as default:
+   ```bash
+   kubectl create namespace fybrik-airbyte-sample
+   kubectl config set-context --current --namespace=fybrik-airbyte-sample
+   ```
+
+1. Create an asset (the `letter-frequency` asset), the policy to access if, and an application that requires this asset:
```

Comment:
it instead of if
What is the policy that's being used in the sample?  Would be good to describe it